internal/pkg/cli/env_package.go

```diff
@@ -154,7 +154,9 @@ func (o *packageEnvOpts) Execute() error {
 	if err != nil {
 		return err
 	}
-
+	if err := deployer.Validate(mft, os.Stderr); err != nil {
```

Comment:
nit: can we use https://github.com/aws/copilot-cli/blob/b6b1c869b0bfafcb9d011aa78a5391e0581854b1/internal/pkg/term/log/log.go#L16
instead of `os.Stderr`. I don't fully remember but these is some funky stuff with colors and Windows https://github.com/mattn/go-colorable/blob/11a925cff3d38c293ddc8c05a16b504e3e2c63be/colorable_windows.go#L102-L118